src/blas/backends/mkl_common/mkl_level1.cxx

```diff
@@ -283,43 +283,71 @@ void swap(sycl::queue& queue, std::int64_t n, sycl::buffer<std::complex<double>,
 }
 
 void iamax(sycl::queue& queue, std::int64_t n, sycl::buffer<float, 1>& x, std::int64_t incx,
-           sycl::buffer<std::int64_t, 1>& result) {
-    RETHROW_ONEMKL_EXCEPTIONS(blas_major::iamax(queue, n, x, incx, result));
+           sycl::buffer<std::int64_t, 1>& result, oneapi::math::index_base base) {
+    RETHROW_ONEMKL_EXCEPTIONS(blas_major::iamax(queue, n, x, incx, result,
+                                                base == oneapi::math::index_base::zero
```

Comment:
For the `base` parameter throughout this file, I think it would be better to add a `get_onemkl_base` function in [common_onemkl_conversion.hpp](https://github.com/uxlfoundation/oneMath/blob/develop/src/include/common_onemkl_conversion.hpp#L36) and use it here (the same way `get_onemkl_transpose` is used [here](https://github.com/uxlfoundation/oneMath/blob/develop/src/blas/backends/mkl_common/mkl_level2.cxx#L25)).

Reply:
fixed in 10760de